MAUI/ProgressBar/custom-content.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: Custom Content in .NET MAUI ProgressBar control | Syncfusion
+description: Learn here all about customcontent support in Syncfusion .NET MAUI ProgressBar control, its elements and more.
+platform: MAUI
+control: ProgressBar
+documentation: ug
+---
+
+# Custom Content in .NET MAUI ProgressBar (Progress Bar)
+
+In the circular progress bar, you can add any view to the center using the `Content` property. 
+
+For example, you can include add, start, or pause button to control the progress. You can also add an image that indicates the actual task in progress or add custom text that conveys how far the task is completed. 
```

Comment:
For example, include add, start, or pause button to control the progress. Also, add an image that indicates the actual task in progress or add custom text that conveys how far the task is completed.

Reply:
done